firestore/query.go

```diff
@@ -687,6 +695,15 @@ func (f filter) toProto() (*pb.StructuredQuery_Filter, error) {
 	if err != nil {
 		return nil, err
 	}
+	if _, ok := arrayOperators[op]; ok {
```

Comment:
Put in a code comment here to explain why this check is needed.

Reply:
Done